### ISSUE TITLE
BrowserFS.InMemory is undefined error in Firefox

### ISSUE THREAD

**Issue by christislord12** [pygame-web/pygbag#129]
On first load of a pygbag project in Firefox it gives a BrowserFS.InMemory error. After reloading the page twice, the error does not happen and the project loads. This error does not occur in Safari or Opera.

<img width="1302" alt="Screenshot 2023-10-26 at 3 30 07 PM" src="https://github.com/pygame-web/pygbag/assets/53404933/83cdd867-fc33-4894-9294-748916015441">

**Comment by Memopentagone** [comment]
yeah can confirm it is happening on Firefox but also appears to be happening on ios with safari

**Comment by pmp-p** [comment]
[long term decision] is to drop BrowserFS and use Panda3D vfs instead. There's no drop-in replacement for it i can see right now ( zipfs+overlayfs ) so not worth the work.

If anyone find something close please drop a note. Basically BrowserFS is a Node FS emulator ( with added complexity) like the one emscripten provides

**Comment by pmp-p** [comment]
They are moving to https://github.com/browser-fs
 and now i don't even know how to load emscripten fs now ( it there's still any)

Javascript bundlers world seems as simple as python packaging so help very welcome here.

 https://github.com/browser-fs/core/issues/22#event-12035817310

**Comment by dr-vortex** [comment]
Hi there, I'm the maintainer of BrowserFS. BrowserFS has been deprecated. Please see the notice in the readme: https://github.com/jvilk/browserfs for more info.

Would you be interested in updating your project to use the updated fork, [ZenFS](https://github.com/zen-fs/core)?

Support for Emscripten is pending on [DefinitelyTyped#69218](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69218).

**Comment by pmp-p** [comment]
> Would you be interested in updating your project to use the updated fork, [ZenFS](https://github.com/zen-fs/core)?

i Would be very interested if support for overlayfs emcripten/OPFS is coming. If not i cannot justify time to make the switch :(

**Comment by dr-vortex** [comment]
@pmp-p,

OPFS is already available with the `WebAccess` backend, provided as part of `@zenfs/dom`. As I mentioned before, I'm waiting for the PR to merge before releasing the Emscripten backend. I expect it to be released one to two weeks after the PR is merged.
